contribution/Killercoda-Kubearmor/kubearmor-installation-view.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+namespace="kubearmor"
```

Comment:
Are we controlling the namespace in which KubeArmor will be installed in Killercoda? If not, then KubeArmor can be installed in any other ns and then this script will not work?

Reply:
Before running the script, during the Helm installation, we specify and create the 'kubearmor' namespace for installing KubeArmor. The script then executes to monitor the status of pods until they are all in the 'Running' state.

Reply:
Okay, but the user can specify their own namespaces in the helm installation, in that case this script will not run.

Reply:
![image](https://github.com/kubearmor/KubeArmor/assets/115368361/58884b34-a4c6-49c2-b942-8fa37fa970b1)

I understand your point, As in the Killercoda case, it's specific to a playground scenario. Users can't run individual commands differently; they need to execute all the commands at once, which are already predefined as shown in the image.

P.S.: I will add that feature in next update. Due to the urgent need, it is currently tailored to the predefined commands.

Reply:
> ![image](https://user-images.githubusercontent.com/115368361/280459043-58884b34-a4c6-49c2-b942-8fa37fa970b1.png)
> 
> I understand your point, As in the Killercoda case, it's specific to a playground scenario. Users can't run individual commands differently; they need to execute all the commands at once, which are already predefined as shown in the image.
> 
Okay, that makes sense.

Reply:
> ![image](https://user-images.githubusercontent.com/115368361/280459043-58884b34-a4c6-49c2-b942-8fa37fa970b1.png)
> 
> I understand your point, As in the Killercoda case, it's specific to a playground scenario. Users can't run individual commands differently; they need to execute all the commands at once, which are already predefined as shown in the image.
> 
Okay, that makes sense.